<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_ArcOptions</class>
 <widget class="QWidget" name="QG_ArcOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>90</width>
    <height>22</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>90</width>
    <height>22</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Arc Options</string>
  </property>
  <widget class="Q3ButtonGroup" name="buttonGroup1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>78</width>
     <height>19</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <widget class="QRadioButton" name="rbNeg">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>2</y>
      <width>36</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Clockwise</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../res/extui/extui.qrc">
      <normaloff>:/extui/dirpos.png</normaloff>:/extui/dirpos.png</iconset>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPos">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>36</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Counter Clockwise</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../res/extui/extui.qrc">
      <normaloff>:/extui/dirneg.png</normaloff>:/extui/dirneg.png</iconset>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="sep1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>2</y>
     <width>6</width>
     <height>19</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::VLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">rs_arc.h</include>
  <include location="local">rs_actiondrawarc.h</include>
 </includes>
 <resources>
  <include location="../../../res/extui/extui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>rbPos</sender>
   <signal>toggled(bool)</signal>
   <receiver>QG_ArcOptions</receiver>
   <slot>updateDirection(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbNeg</sender>
   <signal>toggled(bool)</signal>
   <receiver>QG_ArcOptions</receiver>
   <slot>updateDirection(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
